src/test/kotlin/interactor/GetCityThatMatchTheManagerExpectationsInteractorTest.kt

```diff
@@ -1,87 +1,124 @@
-    fun should_returnCity_when_notAllCountriesIncluded() {
-        // Given not all countries included
-        val countries = listOf("united states", "canada", "iraq")
+    fun `should return exception when countries are valid and prices of meals are invalid`() {
+        // Given that countries' names are valid and prices of meals are invalid
+        val interactor =
+            GetCityThatMatchTheManagerExpectationsInteractor(
+                ManagerExpectationsFakeData.validCountriesAndInvalidPricesOfMeals
+            )
         // When return the city that match manager expectations
-        val result = getCityThatMatchTheManagerExpectationsInteractor.execute(countries)
+        val actualData = Executable {
+            interactor.execute(countries).cityName
+        }
         // Then check the final result
-        assertEquals("Great Falls", result?.cityName)
+        assertThrows(NoSuchElementException::class.java, actualData)
     }
 
     @Test
-    fun should_returnCity_when_allCountriesNotIncluded() {
-        // Given all countries not included
-        val countries = listOf("egypt", "syria", "iraq")
-        // When return empty result
-        val result = getCityThatMatchTheManagerExpectationsInteractor.execute(countries)
+    fun `should return exception when countries and prices of meals are invalid`() {
+        // Given that countries' names  and prices of meals are invalid
+        val interactor =
+            GetCityThatMatchTheManagerExpectationsInteractor(
+                ManagerExpectationsFakeData.invalidData
+            )
+        // When return the city that match manager expectations
+        val actualData = Executable {
+            interactor.execute(countries).cityName
+        }
         // Then check the final result
-        assertNull(null, result?.cityName)
+        assertThrows(IllegalStateException::class.java, actualData)
     }
 
     @Test
-    fun should_returnCity_when_namesOfCountriesIsEmpty() {
-        // Given all empty string instead of countries names
-        val countries = listOf("", "", "")
-        // When return  empty result
-        val result = getCityThatMatchTheManagerExpectationsInteractor.execute(countries)
+    fun `should return exception when empty countries`() {
+        // Given that countries' names  and prices of meals are invalid
+        val interactor =
+            GetCityThatMatchTheManagerExpectationsInteractor(
+                ManagerExpectationsFakeData.emptyData
+            )
+        // When return the city that match manager expectations
+        val actualData = Executable {
+            interactor.execute(countries).cityName
+        }
         // Then check the final result
-        assertNull(null, result?.cityName)
+        assertThrows(IllegalStateException::class.java, actualData)
     }
 
+    companion object{
+        // Given the countries which will search on it
+        val countries = listOf("united states", "canada", "mexico")
```

Comment:
when use variable just in same class set private variable

Reply:
yes sure